<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>naryviz1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutPlotting">
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutPlotting"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButtonGenerate">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>steps</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="spinBoxNumElements">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Element space in</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="spinBoxNumSteps">
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>MDS in</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" colspan="3">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="comboBoxDim">
        <item>
         <property name="text">
          <string>2D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3D</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QPushButton" name="pushButtonCalculate">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QComboBox" name="comboBoxPrecision">
        <item>
         <property name="text">
          <string>low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>high</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Precision</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButtonReplot">
        <property name="text">
         <string>Replot</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBoxLabels">
        <property name="text">
         <string>Labels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBoxCmap">
        <item>
         <property name="text">
          <string>Dark2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dark2_r</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Accent</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>viridis</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>plasma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>hsv</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gnuplot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>gnuplot2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jet</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Colorscale</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
    <addaction name="actionComposition"/>
    <addaction name="actionComposition_with_color_code"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionxyz_position"/>
   </widget>
   <addaction name="menuLoad"/>
   <addaction name="menuSave"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionComposition">
   <property name="text">
    <string>Composition</string>
   </property>
  </action>
  <action name="actionComposition_with_color_code">
   <property name="text">
    <string>Composition with color code</string>
   </property>
  </action>
  <action name="actionxyz_position">
   <property name="text">
    <string>xyz position</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
